### ISSUE TITLE
Increase accuracy of facial recognition model

### ISSUE THREAD

**Issue by theSLWayne** [theSLWayne/tf_facialrecognition#8]
The current model is a Tensorflow image classification model. After training it using [5 Celebrity Faces Dataset](https://www.kaggle.com/dansbecker/5-celebrity-faces-dataset) and the accuracy was ~34%. 

- [x] Change the current model architecture or replace the current model with a different TensorFlow model
- [x] Train the new model
- [x] Test the trained model

**Comment by theSLWayne** [comment]
Done. New validation accuracy: 80%+
